TestResultsQualityGate/task.json

```diff
@@ -18,15 +18,45 @@
   "minimumAgentVersion": "2.209.0",
   "inputs": [
     {
-      "name": "placeholder",
-      "type": "string",
-      "label": "placeholder",
-      "defaultValue": "placeholder",
+      "name": "type",
+      "type": "picklist",
+      "label": "Type",
+      "defaultValue": "Total Passed",
       "required": true,
-      "helpMarkDown": "Type of issues to be evaluated for the threshold."
+      "helpMarkDown": "Type of test results to be evaluated for the threshold.",
+      "options": {
+        "Total Passed": "Total passed tests",
+        "Total Failed": "Total failed tests",
+        "Total Executed": "Total executed tests",
+        "Newly Failed": "Newly failed tests"
+      }
+    },
+    {
+      "name": "threshold",
+      "type": "number",
+      "label": "Threshold",
+      "defaultValue": "0",
+      "required": true,
+      "helpMarkDown": "If the actual total passed/executed tests number is equal to or less than this threshold, then a build is considered unstable or failed . If the actual total or newly failed tests number of test results is equal to or greater than this threshold, then a build is considered unstable or failed. Default value is 0.",
```

Comment:
I think it's not straight forward to understand this desceription. 

@yshi-parasoft how about this way:

The build is considered unstable or failed in the following situations:
	The actual value of the following types is greater than or equal to this threshold.
		- Total failed tests
		- Newly failed tests
	
	The actual value of the following types is less than threshold.	
		- Total passed tests
		- Total executed tests

Reply:
A build is considered unstable or failed when:
- The number of total passed/executed tests is less than this threshold.
- The number of total/newly failed tests is greater than or equal to this threshold.
The default threshold is 0.

Btw, I suggest to remove "actual" in all other descriptions like this.